content/blog/2020/07/2020-07-03-github-checks-api-plugin-coding-phase-1.adoc

```diff
@@ -0,0 +1,109 @@
+---
+layout: post
+title: "GitHub Checks API Plugin Project - Coding Phase 1"
+tags:
+- github
+- api
+- plugins
+- developer
+- gsoc
+- gsoc2020
+author: XiongKezhi
+opengraph:
+  image: /images/gsoc/opengraph.png
+---
+
+This blog post is about our coding phase 1 progress on GSoC project: link:/projects/gsoc/2020/projects/github-checks/[GitHub Checks API Plugin].
+
+The GitHub Checks API is a highly customized way to integrate CI tools to make reports for pull-requests (PRs). 
+It allows users to see CI reports on GitHub pages directly.
+
+image:/images/post-images/2020/07-github-checks-api-plugin-coding-phase-1/github-check-run.png[GitHub Checks API]
```

Comment:
For all images in this blog post: if these are copied from GitHub help pages (i.e. not your screenshots) then you should correctly cite them.